cdm-test/src/test/java/ucar/nc2/util/net/TestHttpRda.java

```diff
@@ -107,7 +127,7 @@ public TestHttpRda(String ds) {
     this.fullUrl = url + ds;
   }
 
-  @Ignore("Failing - HTTP hangs")
+  //@Ignore("Failing - HTTP hangs")
```

Comment:
So this means we're not ignoring this any more? Again, why not just delete the line?